parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -823,6 +823,38 @@ public ParquetFileReader(Configuration conf, Path file, ParquetMetadata footer)
     this.crc = options.usePageChecksumVerification() ? new CRC32() : null;
   }
 
+  /**
+   * @param conf   the Hadoop Configuration
+   * @param file   Path to a parquet file
+   * @param footer a {@link ParquetMetadata} footer already read from the file
+   * @throws IOException if the file can not be opened
+   * @deprecated will be removed in 2.0.0.
```

Comment:
Will this eventually be replaced by another constructor or just not recommended?

Reply:
Oops. Copied the old implementation and did not update the javadoc. Fixed.